common/djangoapps/third_party_auth/colaraz.py

```diff
@@ -0,0 +1,60 @@
+from social_core.exceptions import AuthException
+from django.utils.functional import cached_property
+
+from openedx.core.djangoapps.theming.helpers import get_current_request
+from third_party_auth.identityserver3 import IdentityServer3
+
+class ColarazIdentityServer(IdentityServer3):
+    """
+    An extension of the IdentityServer3 for use with Colaraz's IdP service.
+    """
+    name = "colarazIdentityServer"
```

Comment:
```suggestion
    name = "ColarazIdentityServer"
```

Reply:
It was intentionally made this way, I just mimicked the IdentityServer3 pattern.